flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/rank/AbstractRankFunction.java

```diff
@@ -94,10 +94,18 @@
 	protected long hitCount = 0L;
 	protected long requestCount = 0L;
 
-	AbstractRankFunction(long minRetentionTime, long maxRetentionTime, BaseRowTypeInfo inputRowType,
-			GeneratedRecordComparator generatedSortKeyComparator, BaseRowKeySelector sortKeySelector,
-			RankType rankType, RankRange rankRange, GeneratedRecordEqualiser generatedEqualiser,
-			boolean generateRetraction, boolean outputRankNumber) {
+	AbstractRankFunction(
+		long minRetentionTime,
```

Comment:
Actually these need to indent with 2 tabs

Reply:
OK.